Gemfile

```diff
@@ -93,6 +92,9 @@ group :doc do
   gem 'sdoc', require: false
 end
 
+# needed for /rails/mailers email previews
```

Comment:
And of course also (still) needed for unit testing. Please add that to the comment.